tests/h/schemas/group_test.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import pytest
+# import mock
+
+from h.models.group import (
+    GROUP_NAME_MIN_LENGTH,
+    GROUP_NAME_MAX_LENGTH,
+)
+
+from h.schemas.group import CreateGroupAPISchema
+from h.schemas import ValidationError
+
+
+class TestCreateGroupSchema(object):
+
+    def test_it_raises_if_name_missing(self):
+        schema = CreateGroupAPISchema()
+        with pytest.raises(ValidationError) as exc:
+            schema.validate({})
+
+        # The substring tests here an in following tests are to get
+        # around the occasional literal "u" prefix in Py2 err values
+        assert "is a required property" in str(exc.value)
```

Comment:
You can use the `match` argument to `raises` (a regex) to do this nicer, example:

```python
    def test_it_raises_if_name_missing(self):
        with pytest.raises(ValidationError, match=".*is a required property.*"):
            CreateGroupAPISchema().validate({})
```